node-js-basics-101/node_modules/ws/index.js

```diff
@@ -0,0 +1,15 @@
+/*!
+ * ws: a node.js websocket client
+ * Copyright(c) 2011 Einar Otto Stangvik <einaros@gmail.com>
+ * MIT Licensed
+ */
+
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->